### DISCUSSION TITLE
Can't make a mods folder ONLY for fabric

### DISCUSSION THREAD

**Discussion by commandrod** [FabricMC/fabric-api#1706] [Player support] [answered]
I have recently installed some Fabric mods, and I ran into an issue.
I have some forge mods for 1.8.9 PvP, and 1.7.10.
Forge mods have an option, where you can make a folder inside the mods folder with the version name (Photo attached) and it will only load the 1.8.9 forge mods.
In fabric that is not the case, and I have to remove my fabric mods in order to launch my forge mods and the other way round.
In the attached photo the 1.16.5 folder should contain all of the fabric mods (if you will implement this feature)
Thank you in advance.
![image](https://user-images.githubusercontent.com/57562364/132334821-1a237675-b74d-4c13-b162-9210d98b6e4d.png)

**Comment by warjort** [comment, marked as answer]
Fabric currently has three methods of discovering mods.
* Add them to the classpath when you start fabric/minecraft
* Put them in the mods folder
* Specify the mods to use with a system property ````-Dfabric.addMods=pathToMod1;pathToMod2;pathToMod3````

It doesn't sound like any of these are exactly what you want?
I suggest you raise the issue here:
https://github.com/FabricMC/fabric-loader/issues
